drivers/kw2xrf/kw2xrf_netdev.c

```diff
@@ -625,6 +625,9 @@ static void _isr_event_seq_t(netdev_t *netdev, uint8_t *dregs)
                 netdev->event_callback(netdev, NETDEV_EVENT_TX_COMPLETE);
             }
         }
+        else {
+            netdev->event_callback(netdev, NETDEV_EVENT_TX_COMPLETE);
```

Comment:
I would like to see a comment here too, to document the occurrence.  If I see it correctly, this is the case where `SEQIRQ` fired because 

> This interrupt will assert whenever the Sequence Manager transitions from non-idle to idle state, for any reason

and `CCAIRQ` did not fire because it is disabled by configuration

> CCA Interrupt. A '1' indicates the completion of a CCA operation.